NOTICE.txt

```diff
@@ -0,0 +1,43 @@
+
+The SwiftFoundation Project
+=====================
+
+Please visit the SwiftFoundation web site for more information:
+
+* https://github.com/apple/swift-foundation
+
+Copyright 2023 The SwiftFoundation Project
+
+The SwiftFoundation Project licenses this file to you under the Apache License,
+version 2.0 (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at:
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations
+under the License.
+
+Also, please refer to each LICENSE.txt file, which is located in
+the 'license' directory of the distribution file, for the license terms of the
+components that this product depends on.
+
+---
+
+This product contains derivations of the time calculations from SwiftCertificates.
+
+* LICENSE (Apache License 2.0):
+* https://www.apache.org/licenses/LICENSE-2.0
+* HOMEPAGE:
+* https://github.com/apple/swift-certificates
+
+---
+
+This product contains code to calculate and decompose UNIX timestamps derived from musl libc.
+
+* LICENSE (MIT):
+* https://git.musl-libc.org/cgit/musl/tree/COPYRIGHT
+* HOMEPAGE:
+* https://musl.libc.org
```

Comment:
It doesn't seem standard to have a NOTICE.txt file in our other Swift repos. We have a license, which is reviewed for correctness.

Reply:
We do have a `NOTICE.txt` file in a bunch of repos for when we use other work or have to attribute it. Some examples:

- [swift-certificates](https://github.com/apple/swift-certificates/blob/main/NOTICE.txt)
- [swift-nio](https://github.com/apple/swift-nio/blob/main/NOTICE.txt)
- [swift-cassandra-client](https://github.com/apple/swift-cassandra-client/blob/main/NOTICE.txt)